### DISCUSSION TITLE
Goal 15: Protect, restore and promote sustainable use of terrestrial ecosystems, sustainably manage forests, combat desertification, and halt and reverse land degradation and halt biodiversity loss

### DISCUSSION THREAD

**Discussion by HSCunha** [Project-Resilience/platform#37] [General]
Chat about anything related to Goal 15: [Protect, restore and promote sustainable use of terrestrial ecosystems, sustainably manage forests, combat desertification, and halt and reverse land degradation and halt biodiversity loss](https://github.com/Project-Resilience/platform/blob/main/goals/goal_15.md)

**Comment by danyoungday** [comment]
# B-INTACT
"The Biodiversity Integrated Assessment and Computation Tool (B-INTACT) uniquely seeks to provide a thorough biodiversity assessment of project-level activities in the Agriculture, Forestry and Other Land Use (AFOLU) sector, taking on both a quantitative and a qualitative approach." The tool can be found here: [https://www.fao.org/in-action/epic/ex-act-tool/suite-of-tools/b-intact/en/](https://www.fao.org/in-action/epic/ex-act-tool/suite-of-tools/b-intact/en/). A user can download an Excel spreadsheet where they input the context of their area and the changes they expect to make. Then the spreasheet calculates insights about biodiversity. A user could query the spreadsheet using Python. This project is linked to the EX-ACT project in #35.

## Data

### Context
- Initial setup
  - Country
  - Climate
  - Moisture
  - Elevation
- Land Use
- Infrastructure (roads + railways length in km)
- Extinction risk
- Social value of intact biodiversity

### Actions
- Land-use change
- Infrastructure change
- Extinction risk change
- Ecosystem distribution change

### Outcomes
- Mean species abundance (0-100%), in different phases
- Area of biodiversity loss
- Added social value of biodiversity
- Adjusted MSA for extinction risk

Additionally, qualitative insights can be manually checked based off different tables and guidelines
- Key biodiversity area effect
- Protected area effect
- Share of threatened species affected
- Water stress
- Impact on threatened species
- Risk of alien species
- Impact on water use
- Biodiversity management activities land cost and monetary cost
- Agrobiodiversity practices land and monetary cost
- Improvements land and monetary cost

**Comment by somnrind** [comment]
**Human-wild life conflicts** are increasing day by day and every year it is observed a countless loss of both human lives and animals in addition with great damage in livelihoods and habitats. (sources: https://www.ifaw.org/international/journal/human-wildlife-conflict-kenya, https://www.thehindu.com/news/national/kerala/human-wildlife-conflicts-kerala-loss-of-lives/article69246919.ece, https://timesofindia.indiatimes.com/city/bengaluru/bengaluru-concern-human-wildlife-conflicts-soar-183-elephants-big-cats-spark-fear-near-villages-farms/articleshow/122084666.cms etc. )

A Drone based surveillance system can be developed and deployed in forest adjacent lands which are used for farming and habitats. An AI based software model can be used to detect, monitor and reacts to the herd of animals if found from the video streams captured by Drone Camera in a sensible and responsible manner. The outcome of the models can be shared with foresters and communities on ground in form of alarms. So, instead of physical deterrents drone can reacts with digital deterrents (like loud noises, lights to make illusion) before any encounter happened. 

Now a days drones are becoming very popular in agriculture industry. The same infrastructure may be used to mitigate this conflict.

**Comment by ofrancon** [comment]
For human-wild life conflicts we could use data from:
- satellite images. Animals in forests are not easily detected though. Maybe infrared helps
- wildlife cameras. Maybe there's already a network of cameras we can tap
- noise. Similar to cameras, but with sounds
- news. We can automatically parse news articles about wildlife encounters and the eventual resulting damage, using LLM based multi-agent systems.
- "animal social networks"

On this last source of data, "animal social networks", the idea is to use photos taken by humans when they encounter wild life - there's a good chance they'll take a photo when they do. And publish it to social networks. Then AI can be used to identify specific animals in these pictures (like face recognition, but for animals), and using the GPS data of the picture create a time series of where this individual went over time. That can be used to detect when wild animals get too close to civilization. 

There's been some research in this area. See this excellent presentation by Tanya Berger-Wolf at the NeurIPS 2023 Workshop on Tackling Climate Change with ML: [Trustworthy Al for Biodiversity: Challenges and Opportunities](https://slideslive.com/39014450/trustworthy-al-for-biodiversity-challenges-and-opportunities?ref=folder-130083)

Links:
- [wildme.org](https://www.wildme.org/what-we-do.html) "provides researchers with pioneering Codex platforms with advanced and adaptable computer science tools to collect more animal imagery data, analyze it faster in order to identify animals, and engage the public to continuously optimize solutions in conservation."
- And their [Flukebook](https://www.flukebook.org/) project for what could be a similar project for SDG 14 lifer under water, to track whales paths and avoid collisions with cargo ships
- Also [Imageomics Institute](https://imageomics.osu.edu/)
- And the [BioCLIP](https://imageomics.github.io/bioclip/) model

**Reply by ofrancon** [reply]
In general, efforts made by [Conservation X Labs ](https://www.conservationxlabs.com) look interesting.

**Reply by ofrancon** [reply]
Also:
- [Counting elephants from space](https://www.bath.ac.uk/announcements/counting-elephants-from-space/): satellite images
- [Elephant Listening Project](https://www.elephantlisteningproject.org/): acoustic monitoring

**Reply by somnrind** [reply]
I went through the above links. Satellite image source is a good idea. But, still working to prepare at least some set of Train/Test/Validation data to start over. Also came across below links which may be useful :

https://github.com/satellite-image-deep-learning/techniques/tree/master
https://rastervision.io/
https://www.satimagingcorp.com/applications/natural-resources/forestry/

Thanks!!

**Comment by ofrancon** [comment]
Some animals also carry trackers (GPS)

**Comment by ofrancon** [comment]
That information might not be publicly available, because it could be used by poachers.
